### DISCUSSION TITLE
How to show whole report, to show every similar pictures

### DISCUSSION THREAD

**Discussion by deanAirre** [cleanlab/cleanvision#256] [Q&A] [answered]
That's the question, I cannot find the way to list every similar files so I can delete them

**Comment by deanAirre** [comment]
Found the method written in [ https://www.kaggle.com/code/gpiosenka/useful-image-dataset-tools](here)

**Reply by jwmueller** [reply, marked as answer]
Here's a few resources that should help:

https://cleanvision.readthedocs.io/en/stable/tutorials/tutorial.html#imagelab.issues

https://cleanvision.readthedocs.io/en/stable/tutorials/tutorial.html#Duplicate-sets

You can use: image lab.issues

to see the full list of issues detected for each image.

**Reply by deanAirre** [reply]
Exactly what I need that for some reason didn't appear in my search engine, sorry bout that. Thank you for the answer.
